sdk/couchbase-core/src/crudcomponent.rs

```diff
@@ -691,16 +686,9 @@ impl<
                         async |endpoint: String, vb_id: u16| {
                             orchestrate_memd_client(
                                 self.conn_manager.clone(),
-                                endpoint.clone(),
+                                &endpoint,
                                 async |client: Arc<KvClientManagerClientType<M>>| {
-                                    operation(
-                                        collection_id,
-                                        manifest_rev,
-                                        endpoint.clone(),
-                                        vb_id,
-                                        client,
-                                    )
-                                    .await
+                                    operation(collection_id, manifest_rev, vb_id, client).await
```

Comment:
Why doesn't this need endpoint any more?

Reply:
It was never being used, and we can probably get it from client itself if we need it.